### ISSUE TITLE
Autocomplete not working for the following ZSH Themes

### ISSUE THREAD

**Issue by falky97** [withfig/autocomplete#10]
Fig is not currently compatible with the following ZSH themes:
* https://github.com/romkatv/powerlevel10k

**Comment by falky97** [comment]
**Why does this happen?**

Fig checks which process is currently running in your terminal. We run `ps` then get the last process in the list for a given terminal. 

We don’t show Fig when it’s not bash or zsh, and we show a pared back version of Fig for an ssh session. Otherwise if you’re in a process like nano or vim etc we don’t show Fig

This screenshot shows an example of what happens with the powerlevel10k theme: 
https://files.slack.com/files-pri/T015ZGD7L07-F01DU9Q3ULF/image.png

**Comment by falky97** [comment]
It could potentially be something to do with the [instant-prompt](https://github.com/romkatv/powerlevel10k/blob/master/README.md#instant-prompt) in Powerlevel10k 

More likely it is the git status message as shown in the file shared above

**Comment by dipakparmar** [comment]
@falky97 checked this on my setup with oh-my-zsh. Updating the zsh theme to default **"robbyrussell"** makes working autocomplete somehow but not properly, few issues with cwd (doesn't update) and icons on autocomplete (no icons for folder) 

```
# ZSH_THEME="robbyrussell"
# ZSH_THEME="powerlevel10k/powerlevel10k"
```

Disabling theme fixes this. so maybe it's a conflict with zsh themes.

**Comment by falky97** [comment]
@dipakparmar Yes I noticed those issues too with different themes. However, they seem to go away after restarting iTerm/Terminal...

**Comment by dipakparmar** [comment]
with all theme disabled in iTerm2  just pressing <kbd>space</kbd> getting the blank list

![image](https://user-images.githubusercontent.com/24366206/98484488-aa55fa00-21c4-11eb-8b3d-6b25da39b7b7.png)

**Comment by falky97** [comment]
Usually that happens in debug mode… Would suggest quitting Fig and restarting

Brendan Falk
Co-founder & CEO @ Fig
brendan@withfig.com
(857) 800-1383
